docs/getting-started/starter-guide.md

```diff
@@ -0,0 +1,58 @@
+# Starter Guide
+
+Welcome to **Theatria**, your gateway to an immersive Minecraft experience! This guide will help you get started, from connecting to our server to thriving in our unique world.
+
+---
+
+## Step 1: Connecting to Theatria
+
+1. **Launch Minecraft**: Open the Minecraft launcher and ensure you have the latest version of Minecraft Java Edition installed.
+2. **Add the Server**:
+    - Click on **Multiplayer**.
+    - Select **Add Server**.
+    - Enter the following details:
+        - **Server Name**: Theatria
+        - **Server Address**: `mc.playtheatria.com`
+    - Save and click **Join Server** to connect.
+
+---
+
+## Step 2: Rules and Expectations
+
+To ensure everyone has a great time, please follow our community guidelines:
+
+- **Be respectful**: Treat other players and staff with kindness.
+- **No griefing or stealing**: Protecting player builds and items is our priority.
+- **Keep chat friendly**: Avoid offensive language and spam.
+- **Play fair**: No cheating, exploiting, or using unauthorized mods.
+
+You can view the full rules on our [Rules Page](../rules-policies/rules.md).
+
+---
+
+## Step 3: Start Exploring Theatria
+
+Once you join the server, you will start in our **Welcome Hub** in **The Ark**. 
+
+### Essential Commands:
+- `/spawn`: Return to the Welcome Hub.
+- `/claim`: Claim a chunk of land. Learn more about [Land Claiming](../gameplay-features/land-claiming.md).
+- `/sethome [name of home]`: Set your current location as your home. As players rank up, they can save more homes.
+- `/home [name of home]`: Teleport to your saved home.
+- `/warps`: Find and visit community landmarks and player-submitted points of interest.
+
+You can find a full list of commands on our [Commands Page](../gameplay-features/commands.md).
+
+---
+
+## Step 4: Getting Help
+
+Need assistance? Here’s how to find support:
+
+- **Ask in Chat**: Our community and staff are here to help.
+- **Visit the Wiki**: Explore detailed guides here in [Theatria Wiki](https://docs.playtheatria.com/).
+- **Join the Discord**: Connect with us at [discord.gg/theatria](https://discord.gg/theatria) for updates and live support.
```

Comment:
This discord link needs to be updated.
This one should work:
https://discord.gg/SHgauw8eN8